docs/tutorial/kymotracking.rst

```diff
@@ -640,3 +638,34 @@ in :doc:`Population Dynamics </tutorial/population_dynamics>`.
     The `min_observation_time` and `max_observation_time` arguments to the underlying :class:`~lumicks.pylake.DwelltimeModel` are set automatically by this method.
     The minimum length of the tracks depends not only on the pixel dwell time but also the specific input parameters used for the tracking algorithm.
     Therefore, in order to estimate these bounds, the method uses the shortest track time and the length of the experiment, respectively.
+
+.. _global_analysis:
+
+Global analysis
+---------------
+
+Sometimes, we want to combine tracking results from multiple Kymographs to determine biophysical parameters with increased precision.
+We can do this, by simply adding :class:`~lumicks.pylake.kymotracker.kymotrack.KymoTrackGroup` instances together.
+We'll demonstrate this functionality using multiple sections on a single :class:`~lumicks.pylake.kymo.Kymo`, but it generalizes to kymographs with different acquisition parameters::
+
+    tracks1 = lk.filter_tracks(lk.track_greedy(kymo, "green", rect=[[127, 9], [162, 26]]), 4)
+    tracks2 = lk.filter_tracks(lk.track_greedy(kymo, "green", rect=[[175, 9], [200, 26]]), 4)
+
+    multiple_groups = tracks1 + tracks2
+    multiple_groups.plot()
+
+.. image:: figures/kymotracking/multi_group.png
+
+The API for the different methods is identical, requiring no changes to your downstream analysis for the single Kymograph case.
+For instance, one can compute a dwell time analysis by invoking::
+
+    multi_dwell = multiple_groups.fit_binding_times(n_components=2)
+    print(multi_dwell.lifetimes)  # list of bound lifetimes
+    print(multi_dwell.amplitudes)  # list of fractional amplitudes for each component
+
+This will take into account the impact of the different line times on the dwell time analysis (see :doc:`Population Dynamics </tutorial/population_dynamics>` for more information).
```

Comment:
Right, that's true. I'll remove it for now, but in the future, it might be worth mentioning somewhere. Perhaps in the new dwelltime docs. It's something that as an advanced user I would wonder about when reading that the minimum and maximum time get taken into account.

Reply:
yeah I definitely think we should mention it somewhere. I just don't think there's the proper context here and it's a bit distracting. Yeah probably in the dwelltime docs is best .... for later